spring-integration-gcp/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xmlns="http://maven.apache.org/POM/4.0.0"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>spring-cloud-gcp</artifactId>
+		<groupId>org.springframework.cloud</groupId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+	<name>Spring Integration GCP</name>
+	<description>Integration modules for Google Cloud Platform</description>
+	<artifactId>spring-integration-gcp</artifactId>
+	<organization>
```

Comment:
@artembilan  I think I get the gist of it, but it will probably mean this is a separate module. @garyrussell We may need to move it away from this repo then, otherwise any maven command ran at the root would not pick this. I'll do some research around ways we could have it on the same repo. I'm opening an issue for this if you are ok with that